notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-data2vec.ipynb

```diff
@@ -0,0 +1,682 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    print(f"[GT]: {dataset[0]['text']}")
</pre><p>For clarity it would be nice to use the same padding to have the same words one above the other (https://stackoverflow.com/a/5676884/1624463). It's easier to compare them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/785/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>added, thank you</p>

Reply:
<p>Something is wrong here. You should use the same value for all lines e.g. "&gt;20". Run the notebook and see the difference</p><p> </p>

Reply:
this value means length of string in symbols and alignment (left <, right >, center = ), due to difference in transcription these lines have different length, so it is a reason why values are different

Reply:
but I managed it just using spaces :)